README.md

```diff
@@ -98,6 +100,7 @@ The **MarkdownTOC** plugin is rich on features and customization, useful for bot
 - [Customizable list bullets in TOC](#customizable-list-bullets-in-toc)
 - [Specify custom indentation prefix](#specify-custom-indentation-prefix)
 - [Preserve images in headings](#preserve-images-in-headings)
+- [Excluded heading](#excluded-heading)
```

Comment:
Rewrite to "Excluded headings", since we are describing the feature in generically